### ISSUE TITLE
open unable to open file (in target "Tests iOS" in project "SDWebImage Tests")

### ISSUE THREAD

**Issue by PROCW** [SDWebImage/SDWebImage#3253]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [x] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | macos
 Platform Version        | 11.2 Big Sur
 SDWebImage Version      | lastestversion
 Integration Method      | manually
 Xcode Version           | Xcode 12.5
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

I downloaded the source from master branch, and open the PSDWriterExample and sign with my apple development account.
Next, I build the source,  checked the [Target Membership(Tests Mac)], but error is showed.
[open unable to open file (in target "Tests iOS" in project "SDWebImage Tests")]

**Comment by kinarobin** [comment]
11.2 Big Sur  Xcode 12.5   

![截屏2021-08-03 下午6 15 03](https://user-images.githubusercontent.com/29303030/127999253-5615352d-b0cc-4130-9d45-bd76d804efa5.png)

**Comment by kinarobin** [comment]
@PROCW  If the issue has not been solved, feel free to reopen.

**Comment by Deco354** [comment]
Can we reopen this? I've the same issue.
<img width="1140" alt="image" src="https://user-images.githubusercontent.com/18557720/181497808-ed16fba4-fd66-49e1-9421-44a28c0910cd.png">
